Shorten the "Technologies" article in pedia...

Ouaz opened the pull request:
... by making a dedicated article about "Researching Technologies" in *Guides*

- Add a sentence about Fighter Damage ignoring Shields in "Damage" and "Shields" pedia articles
- Add plural form to meter types "Shields" and "Max Shields" to prevent grammatical errors in pedia articles which have [[metertype METER_SHIELD]] pedia links (as the previous SHIELDS_TITLE value was "Shields" and not "Shield")
- Add some pedia links, fix a few typos and formatting errors, remove some unnecessary triple quotes

As usual, needs a proofreading, as my written english can be broken. :p

Ouaz committed:
Shorten the "Technologies" article in pedia...

... by making a dedicated article about "Researching Technologies" in *Guides*

dbenage-cx reviewed (changes requested):
Unrelated changes should probably be in a separate commit

LGM-Doyle reviewed (commented):
Looks good.  I added two minor suggestions.

Ouaz committed:
Add a sentence about Fighter Damage ignoring Shields in "Damage" and "Shields" pedia articles

Ouaz committed:
Add plural form to meter types "Shields" and "Max Shields"

...to prevent grammatical errors in pedia articles which have [[metertype METER_SHIELD]] pedia links (as the previous SHIELDS_TITLE value was "Shields" and not "Shield")

Ouaz committed:
Various additions/fixes and some cleaning in EN stringtables

Add some pedia links, fix a few typos and formatting errors, remove some unnecessary triple quotes
